.github/CODEOWNERS

```diff
@@ -525,6 +536,11 @@ sdk/trafficmanager/Microsoft.Azure.Management.TrafficManager/            @tmsupp
 # ServiceLabel: %Notification Hub %Service Attention
 /sdk/notificationhubs/Microsoft.Azure.Management.NotificationHubs/            @tjsomasundaram
 
+# ServiceLabel: %Mixed Reality %Service Attention
```

Comment:
```suggestion
```

There can only be one rule for a given label, so the "ServiceLabel" bit isn't needed.